backend/tests/requests.http

```diff
@@ -0,0 +1,26 @@
+POST http://127.0.0.1:5000/receive HTTP/1.1
```

Comment:
This looks like part of a test, but there's no actual test using this file. Did you forget to check it in?

Either add missing test files or remove this and x.png file. Obviously, having a test is better than having none.
Note there are 5 unit-tests on master for Flask already. They're very basic, but perhaps you could
rebase the branch and expand on the very basic framework that's already there? See [test_satnogs.py](https://github.com/gut-space/satnogs/blob/master/backend/tests/test_satnogs.py).

Reply:
It isn't test. It is example of HTTP message which should be passed to receive action.  
I use RestClient Visual Studio Code extension to send it. It is simply and may be requests may be commited to repo.
Maybe I should move this file to "request" directory or similar?

Reply:
Moved to backend/examples/